src/arena.py

```diff
@@ -1 +1,146 @@
-WORKS = True
+import random
+import re
+from dataclasses import dataclass
+from typing import List
+
+
+class Combatant:
+    def __init__(self, name: str, health: int, damage: str):
+        self.name = name
+        self.health = health
+        self.__damage = self.Damage(damage)
+        self.last_damage = 0
+        self.last_health = health
+
+    @property
+    def damage(self) -> str:
+        return self.__damage.to_string()
```

Comment:
In Python you can define `__str__` method that will be invoked when you do `str(obj)`